pages/companies/[slug].vue

```diff
@@ -194,38 +194,37 @@ const latestComp = computed(() => {
   return sortCollection(company.compensations, ref("createdAt"), sortOrder.value)[0];
 });
 
-const jsonLd = 
-{
+const jsonLd = {
   "@context": "https://schema.googleapis.com/",
   "@type": "OccupationAggregationByEmployer",
-  "name": "Software Engineer",
-  "mainEntityOfPage": {
+  name: "Software Engineer",
```

Comment:
share linter pls so this doesn't happen again

Reply:
With this setup you should be good to go.
https://github.com/Suge-Team/salary-web/pull/38#issuecomment-1455145510